### ISSUE TITLE
[BUG] Depressurizer only registers items with an ID under one million

### ISSUE THREAD

**Issue by LearnTheNext** [Depressurizer/Depressurizer#242]
**Describe the bug**
Steam items not have IDs greater than one million and, although the Depressurizer API is capable of returning this data, the interface only acknowledges items with an ID of one million or lower (0-1000000)

**To Reproduce**
1. Add any item with an ID >1M to Steam library (or confirm in sharedconfig.vdf file that one is already present)
2. Put Steam in offline mode and categorize the new item to add it to the sharedconfig.vdf file
3. Open Depressurizer
4. Update Game List and Import Steam Config
5. New item does not appear

**Expected behavior**
Items should appear in Depressurizer regardless of ID

**Comment by mvegter** [comment]
I was able to successfully import and categorize the `F1® 22` game which has an id of  `1_692_250`

**Comment by LearnTheNext** [comment]
You're correct; I was having a different issue where most of my library was not registering in Depressurizer. Unregistered items all happened to have an ID < 1M. When I saw the ID range in the Database Editor was limited to items < 1M, I made an assumption. I'm still having random items not show up even after using my known workarounds, but haven't been able to determine what they have in common to open a proper report.
